shared/oval/disable_interactive_boot.xml

```diff
@@ -1,34 +1,74 @@
 <def-group>
-  <definition class="compliance" id="disable_interactive_boot" version="2">
+  <definition class="compliance" id="disable_interactive_boot" version="3">
     <metadata>
-      <title>Disable Interactive Boot</title>
+      <title>Verify that Interactive Boot is Disabled</title>
       <affected family="unix">
         <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
       </affected>
       <description>The ability for users to perform interactive startups should
       be disabled.</description>
-      <reference source="JL" ref_id="20140407" ref_url="test_attestation" />
+      <reference source="JL" ref_id="RHEL7_20160613" ref_url="test_attestation" />
+      <reference source="JL" ref_id="FEDORA23_20160613" ref_url="test_attestation" />
     </metadata>
-    <criteria>
-      <criterion test_ref="test_disable_interactive_boot" />
-      <!-- This needs to be enhanced yet to properly cover case of 'systemd.confirm_spawn=1'
-           use case on RHEL-7 and Fedora per:
-             https://rhsummit.files.wordpress.com/2014/04/summit_demystifying_systemd1.pdf Page #20
-           SSG Upstream ticket to come yet -->
+    <criteria operator="OR">
+      <criterion test_ref="test_disable_interactive_boot_grub_cmdline_linux"
+      comment="Check systemd.confirm_spawn=(1|yes|true|on) not in GRUB_CMDLINE_LINUX" />
+      <criteria operator="AND">
+        <criterion test_ref="test_disable_interactive_boot_grub_cmdline_linux_default"
+        comment="Check systemd.confirm_spawn=(1|yes|true|on) not in GRUB_CMDLINE_LINUX_DEFAULT" />
+        <criterion test_ref="test_bootloader_recovery_disabled"
+        comment="Check GRUB_DISABLE_RECOVERY=true in /etc/default/grub" />
```

Comment:
The issue is ```GRUB_CMDLINE_LINUX``` value is used only when ```GRUB_DISABLE_RECOVERY``` is not enabled. If it is, ```GRUB_CMDLINE_LINUX_DEFAULT``` value is honored instead. This has been discussed earlier within som PR related to ```nousb``` grub.cfg kernel argument IIRC.